src/HardhatChainlink.ts

```diff
@@ -1540,7 +1540,7 @@ class DRConsumer {
   }
 
   public async requestData(
-    directRequestConsumerAddress: string,
```

Comment:
any critical reason for this change? i prefer the old naming - longer but not ambiguous!

Reply:
That's true, thank you for mentioning it, I'll rename it back.